package.json

```diff
@@ -18,7 +18,10 @@
     "lic_transitive": "npx @adsk/adsk-npm-license-puller --path . --app-name 'dynamo-home' --verbose --about-box ./license_output/about-box_transitive.html --about-box-type desktop --transitive --year $(date) --paos ./license_output/paos_transitive.csv",
     "generate_license": "npm run lic_direct && npm run lic_transitive",
     "test:unit": "NODE_ENV=test jest tests/App.test.js",
-    "test:e2e": "playwright test tests/e2e.test.js"
+    "test:e2e": "playwright test tests/e2e.test.js",
+    "lic_direct": "npx @adsk/adsk-npm-license-puller --path . --app-name 'home-page' --verbose --about-box ./license_output/about-box_direct.html --about-box-type desktop --year $(date \"+%Y\") --paos ./license_output/paos_direct.csv",
+    "lic_transitive": "npx @adsk/adsk-npm-license-puller --path . --app-name 'home-page' --verbose --about-box ./license_output/about-box_transitive.html --about-box-type desktop --transitive --year $(date \"+%Y\") --paos ./license_output/paos_transitive.csv",
+    "generate_license": "npm run lic_direct && npm run lic_transitive"
```

Comment:
These changes should be reverted

Reply:
No, I don't think so .. I just checked again and pulled origin master and everything looks ok.

I will remove these manually.